admin/public/js/components/host_stats.jsx

```diff
@@ -0,0 +1,45 @@
+define(['pubsub', 'react', 'jsx!components/charts'], function (PubSub, React, MovingLineChart) {
+
+  var xdomain_length = 300;
+  var DATA_INTERVAL = 5000;
```

Comment:
`defaultProps` is a good place for these, allowing for more configurable and reusable components (in general, this is fine).